homeassistant/components/wittiot/sensor.py

```diff
@@ -0,0 +1,356 @@
+    ),
+    WittiotSensorEntityDescription(
+        key="feellike",
+        translation_key="feellike",
+        native_unit_of_measurement=UnitOfTemperature.FAHRENHEIT,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key="dewpoint",
+        translation_key="dewpoint",
+        native_unit_of_measurement=UnitOfTemperature.FAHRENHEIT,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key="tf_co2",
+        translation_key="tf_co2",
+        native_unit_of_measurement=UnitOfTemperature.FAHRENHEIT,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key="humidityin",
+        translation_key="humidityin",
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.HUMIDITY,
+    ),
+    WittiotSensorEntityDescription(
+        key="humidity",
+        translation_key="humidity",
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.HUMIDITY,
+    ),
+    WittiotSensorEntityDescription(
+        key="humi_co2",
+        translation_key="humi_co2",
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.HUMIDITY,
+    ),
+    WittiotSensorEntityDescription(
+        key="baromrelin",
+        translation_key="baromrelin",
+        native_unit_of_measurement=UnitOfPressure.INHG,
+        device_class=SensorDeviceClass.PRESSURE,
+    ),
+    WittiotSensorEntityDescription(
+        key="baromabsin",
+        translation_key="baromabsin",
+        native_unit_of_measurement=UnitOfPressure.INHG,
+        device_class=SensorDeviceClass.PRESSURE,
+    ),
+    WittiotSensorEntityDescription(
+        key="winddir",
+        translation_key="winddir",
+        icon="mdi:weather-windy",
+        native_unit_of_measurement=DEGREE,
+    ),
+    WittiotSensorEntityDescription(
+        key="windspeedmph",
+        translation_key="windspeedmph",
+        icon="mdi:weather-windy",
+        native_unit_of_measurement=UnitOfSpeed.MILES_PER_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key="windgustmph",
+        translation_key="windgustmph",
+        icon="mdi:weather-windy",
+        native_unit_of_measurement=UnitOfSpeed.MILES_PER_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key="daywindmax",
+        translation_key="daywindmax",
+        icon="mdi:weather-windy",
+        native_unit_of_measurement=UnitOfSpeed.MILES_PER_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WittiotSensorEntityDescription(
+        key="uv",
+        translation_key="uv",
+        native_unit_of_measurement="Index",
```

Comment:
```suggestion
        native_unit_of_measurement="index",
```
I might even argue that index isn't a unit of measurement. I have 1 index, 2 indexes...